docs/src/conf.py

```diff
@@ -294,7 +294,9 @@ def _dotv(version):
     "collapse_navigation": True,
     "navigation_depth": 3,
     "show_prev_next": True,
-    "navbar_align": "content",
```

Comment:
Ah that explains why we were already left-aligned! I wanted to explicitly set this so that there was room enough for all 6 headings. I suppose it's a trade-off with how neat it looks.

Reply:
For now lets get this merged, I have a notification setup for the **pydata-sphinx-theme** releases and can check to see if the `navbar_align` is fixed and looks ok when available.